cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/diff/change/parsing/NewObjectParsingStrategy.java

```diff
@@ -14,16 +16,23 @@
  */
 public class NewObjectParsingStrategy extends AbstractParsingStrategy {
 
+    private static final Log log = Log.getLog(NewObjectParsingStrategy.class);
+
     @Override
     public List<Class<? extends Change>> getMatchingTypes() {
         return Arrays.asList(NewObject.class);
     }
 
     @Override
     protected List<CfChange> doParse(Change change) {
-        return Arrays.asList(new CfNewObject(change.getAffectedObject().get(),
+        log.verbose("Parsing change type", change.getClass().getSimpleName(), "to custom change type",
```

Comment:
Should be `info`.

Reply:
I would recommend to use `debug` logging here because this log message could occur very often and does not provide valuable information to the user